Product/Production/Adapters/General/CONNECTAdminGUI/pom.xml

```diff
@@ -16,6 +16,24 @@
     <dependencies>
         <!-- Java EE APIs -->
         <dependency>
+            <groupId>org.connectopensource</groupId>
+            <artifactId>PatientDiscoveryCore</artifactId>
+            <version>4.5.0-SNAPSHOT</version>
+            <type>jar</type>
+        </dependency>
+        <dependency>
+            <groupId>org.connectopensource</groupId>
+            <artifactId>DocumentQuery_a0</artifactId>
+            <version>4.5.0-SNAPSHOT</version>
+            <type>war</type>
+        </dependency>
+        <dependency>
+            <groupId>org.connectopensource</groupId>
+            <artifactId>DocumentQueryCore</artifactId>
+            <version>4.5.0-SNAPSHOT</version>
+            <type>jar</type>
+        </dependency>
```

Comment:
DocumentQuery_a0 should not be used as a dependency; for the other two, please do the following:

- Remove "<type>jar</type>" (this is default)
- Change the version to "${project.parent.version}"
- Move from under "Java EE APIs" to "CONNECT Modules"